src/libs/SidebarUtils.ts

```diff
@@ -447,8 +455,60 @@ function getOptionData({
 
     return result;
 }
+function getReportBeginningOfChatHistoryMessage(report: OnyxEntry<Report>): string {
+    if (ReportUtils.isThread(report)) {
+        return Localize.translateLocal('report.noActivityYet');
+    }
+
+    const welcomeMessage = ReportUtils.getWelcomeMessage(report);
+    if (ReportUtils.isPolicyExpenseChat(report)) {
+        if (report?.description) {
+            return parseHtmlToText(report.description);
+        }
+        return `${welcomeMessage.phrase1} ${ReportUtils.getDisplayNameForParticipant(report?.ownerAccountID)} ${welcomeMessage.phrase2} ${ReportUtils.getPolicyName(report)} ${
+            welcomeMessage.phrase3
+        }`;
+    }
+
+    if (ReportUtils.isChatRoom(report)) {
+        if (report?.description) {
+            return parseHtmlToText(report.description);
+        }
+        return `${welcomeMessage.phrase1} ${welcomeMessage.showReportName ? ReportUtils.getReportName(report) : ''} ${welcomeMessage.phrase2 ?? ''}`;
+    }
+
+    if (ReportUtils.isSelfDM(report) || ReportUtils.isSystemChat(report)) {
+        return `${welcomeMessage.phrase1}`;
+    }
+
+    const participantAccountIDs = ReportUtils.getParticipantsAccountIDsForDisplay(report);
+    const isMultipleParticipant = participantAccountIDs.length > 1;
+    const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(
+        OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, allPersonalDetails),
+        isMultipleParticipant,
+    );
+    const displayNamesWithTooltipsText = displayNamesWithTooltips
+        .map(({displayName, pronouns}, index) => {
+            const formattedText = !pronouns ? displayName : `${displayName} (${pronouns})`;
+
+            if (index === displayNamesWithTooltips.length - 1) {
+                return `${formattedText}.`;
+            }
+            if (index === displayNamesWithTooltips.length - 2) {
+                return `${formattedText} ${Localize.translateLocal('common.and')}`;
+            }
+            if (index < displayNamesWithTooltips.length - 2) {
+                return `${formattedText},`;
+            }
+
+            return '';
+        })
+        .join(' ');
```

Comment:
In `ReportWelcomeText` this logic is conditioned by ` !(isChatRoom || isPolicyExpenseChat || isSelfDM || isInvoiceRoom || isSystemChat); ` Let's keep the same condition